connector_node/risingwave-source-test/src/test/java/com/risingwave/connector/PostgresSourceTest.java

```diff
@@ -0,0 +1,186 @@
+package com.risingwave.connector;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+import com.risingwave.proto.ConnectorServiceProto;
+import io.grpc.Grpc;
+import io.grpc.InsecureChannelCredentials;
+import io.grpc.Server;
+import io.grpc.ServerBuilder;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Iterator;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.locks.Condition;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+import org.junit.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.utility.MountableFile;
+
+public class PostgresSourceTest {
+    private static final Logger logger =
+            LoggerFactory.getLogger(PostgresSourceTest.class.getName());
+
+    private static final PostgreSQLContainer<?> pg =
+            new PostgreSQLContainer<>("postgres:12.3-alpine")
```

Comment:
What does the `alpine` mean in the version number?

Reply:
to be honest I randomly picked an image...
from stack overflow:
> Alpine is a much smaller version of Linux, it results in a smaller container than the full postgres image.
https://stackoverflow.com/questions/62333176/docker-difference-postgres12-from-postgres12-alpine